android/capacitor/src/main/java/com/getcapacitor/Bridge.java

```diff
@@ -883,4 +883,8 @@ public BridgeWebViewClient getWebViewClient() {
     public void setWebViewClient(BridgeWebViewClient client) {
         this.webViewClient = client;
     }
+
+    public void registerGeolocationPlugin(Plugin geolocationPlugin) {
```

Comment:
@dwieeb I added this method to the bridge so that it's possible to provide a Geolocation plugin for the `onGeolocationPermissionsShowPrompt` method in `BridgeWebChromeClient` to use. I'm unsure if we want to leave it at this, or make this method available on the `Plugin` class instead so that plugins don't need to touch the Bridge to do this. Thoughts?

Reply:
Would it make more sense to have it be more of an event-based design?